### ISSUE TITLE
Using Expo for BlossomApp

### ISSUE THREAD

**Issue by jmc-codeless** [hrc2/blossom-public#32]
I loaded the Expo app on my iPhone and created an account.
I cannot seem to get npm to properly install exp (more current is expo-cli) on my Windows 10 laptop.  After the script runs and I cd to the BlossomApp directory, for both `epo publish` and `expo-cli publish` , the script responds with:
Error: node_modules directory is missing.  Please run `npm install` in your project directory.
Couldn't start project.  Please fix the errors and restart the project.
I have screen shots of the npm install script and can email the png file if this link doesn't work.
![image](https://user-images.githubusercontent.com/52114160/67153812-bcad7f00-f2b5-11e9-9a36-7863407b8eca.png)

**Comment by jmc-codeless** [comment]
Hello ... Can anyone please give me some guidance here?

**Comment by psychomugs** [comment]
Hey jmc,

Did you run `npm install` in the directory before `expo-cli publish`?

**Comment by jmc-codeless** [comment]
Yes.  I typed it into the project directory, Blossom-Public-Main, and into the subfolder BlossomApp.  The screen shot I left with the issue shows the response I got (have to zoom in).  It was about the same in both folders locations.  I do not know what it is supposed to look like.  But, when I type ‘expo-cli publish’ in the BlossomApp subfolder, the same Warning and Error message sequences appear.

**Comment by psychomugs** [comment]
Hi jmc,

I think if you ran `npm install`, e.g. not `npm install -g expo-cli`, it should've created the `node_modules` folder that the error is saying is missing.

**Comment by jmc-codeless** [comment]
Hi psychomugs,
I may have gone a little overboard and missed a few steps.  I uninstalled python 2.7 and python 3.7 and installed python3.8 (& changed system PATH).  I downloaded and unzipped #33 package from git.  Ran 'pip install -r requirements.txt'   Created a venv (BTW, my top-level dir is 'blossom-public-master').  went to subdir BlossomApp and ran 'npm install'.  I got a lot of warning and errors related to ENOENT missing; npm could not find the file.  Here's the screen shot.  I have the logfile if that would help.
![Warns for no ENOENT](https://user-images.githubusercontent.com/52114160/68997207-7c67ff00-0869-11ea-8d6f-632d8adc9c5b.png)

**Comment by psychomugs** [comment]
Hey jmc,

Unfortunately this seems a little beyond our expertise, as I don't have a Windows machine to test this problem on. We're also getting a bit preoccupied with end-of-semester things so you'll probably find answers faster if you Google the npm-related issues.

Related to #33 , try renaming the top-level directory to just `blossom` and see if it resolves any issues. This might require you to recreate your `virtualenv` though.

**Comment by jmc-codeless** [comment]
Hi Michael,Understood.  Is there any suggestion about the installation of expo-cli requiring Python 2.7 and not being compatible with Python 3x?-----Original Message-----

**Comment by psychomugs** [comment]
Hey jmc,

The current codebase is designed to work with Python3, and I've been running it with 3.5 or 3.7, so it might be a platform-specific issue. Did you pull the latest changes from ~12 days ago?
